development/components/form/types-reference/formatted-textarea.md

```diff
@@ -15,7 +15,17 @@ Enables TinyMCE text editor on TextareaType.
 
 ## Required Javascript components
     
-None.
+To use with twig and new prestashop admin UI Kit. You have to init the component :
```

Comment:
To make it work in the PrestaShop UI Kit, you need to call the `TinyMCEEditor` js component :